prime-router/src/main/kotlin/Receiver.kt

```diff
@@ -245,12 +245,6 @@ open class Receiver(
      * Validate the object and return null or an error message
      */
     fun consistencyErrorMessage(metadata: Metadata): String? {
-        if (conditionFilter.isNotEmpty() || mappedConditionFilter.isNotEmpty()) {
```

Comment:
Why was this removed?

Reply:
> Why was this removed?

nw these kind of check is done in schema validation